runtime/platform/log.cpp

```diff
@@ -18,7 +18,7 @@ namespace executor {
 namespace internal {
 
 /// Maximum length of a log message.
-static constexpr size_t kMaxLogMessageLength = 256;
+__ET_UNUSED static constexpr size_t kMaxLogMessageLength = 256;
```

Comment:
This isn't always unused, only when logging is disabled. Instead of annotating it here, please move the definition down into `vlogf()` inside the `#ifdef`, then it will only be defined when it's actually used. Or guard this definition with `#if ET_LOG_ENABLED` so it's only defined when it's actually used.

Reply:
Done.